forge/test/mlir/test_loss.py

```diff
@@ -62,3 +61,32 @@ def test_cross_entropy_loss(prediction_shape):
     torch_loss_out = torch_loss(prediction, target)
 
     assert torch.allclose(torch_loss_out, forge_loss_out[0], rtol=11e-3)
+
+
+@pytest.mark.parametrize(
+    "prediction_shape",
+    [
+        (2, 2),
+        (3, 5),
+        (32, 32),
+        (33, 127),
+        (128, 20),
+        (128, 128),
+    ],
+)
+def test_mse_loss(prediction_shape):
+    forge_loss = forge.op.loss.MSELoss("mse_loss", reduction="avg")
```

Comment:
We should test all reductions.
Also, nit: it makes sense to me to use same reduction names as torch.